src/log/file_appender.cpp

```diff
@@ -82,13 +82,16 @@ namespace fc {
                    out.flush();
                    out.close();
                }
+               string rez;
+               read_file_contents(link_filename, rez);
```

Comment:
The default behaviour is fixed now, I've checked it. Test please too on your side. Awaiting your results ...

Reply:
no, I didn't forgot to push my new code :-) read_file_contents copy just piece of data logs which depends on rotation intervals from config file. In current solution as you can see we won't have huge 100 GB log files or even just a few GB files won't be, depends also on file rotation intervals from config.

Reply:
What if the rotation interval is configured to a day?